### PR TITLE
GDB-8759: Implement operations status component in the WB

### PR DESCRIPTION
**Pull request by boyan-tonchev**
## What
- Implement a component that displays the status of active operations, such as running queries, imports, backup, and restore processes.
 -Removed the component for displaying running import, update and queries operations.

## Why
We aim to make it easy for users to understand when there are ongoing operations within the system.

## How
Implement a global component that fetches data every 2 seconds and displays information if there is at least one active operation.

### CODE REVIEW

File: src/js/angular/core/directives/operations-statuses/operations-statuses.directive.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        /**
+         * Getters for monitoring view url depending on <code>operationGroupType</code>.
+
+         * @param {string} operationGroupType - the value must be one of {@see OPERATION_GROUP_TYPE} options.
+         *
+         * @return {string} the url of the monitoring view.
+         */
+        scope.getOperationUrl = (operationGroupType) => {
+            if (OPERATION_GROUP_TYPE.QUERIES_OPERATION === operationGroupType) {
+                return 'monitor/queries';
```
Comment (yordanalexandrov) [inline src/js/angular/core/directives/operations-statuses/operations-statuses.directive.js:39]: Maybe extract them to a `RedirectUrl` constant on top.

File: src/js/angular/controllers.js [hunk 2]
```diff
@@ -931,6 +840,50 @@ function mainCtrl($scope, $menuItems, $jwtAuth, $http, toastr, $location, $repos
 
         return $filter('date')(time, ("'" + $translate.instant('timestamp.on') + "' yyyy-MM-dd '" + $translate.instant('timestamp.at') + "' HH:mm"));
     };
+
+    // =========================
+    // Private functions
+    // =========================
+    const fibonacciGenerator = SequenceGeneratorUtil.getFibonacciSequenceGenerator();
+    const updateActiveOperations = () => {
```
Comment (svilenvelikov) [inline src/js/angular/controllers.js:848]: I wonder, shouldn't these be in the directive itself and not here

Comment (boyan-tonchev) [reply src/js/angular/controllers.js:848]: When I implemented it, I thought about the directive as a component for visualizing a list of operations. The main controller is the client of this component, responsible for fetching and managing the data.

File: src/js/angular/core/directives/operations-statuses/operations-statuses.directive.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
+import {OPERATION_GROUP_TYPE} from "../../../models/monitoring/operations/operation-group";
+import {OPERATION_STATUS} from "../../../models/monitoring/operations/operation-status";
+import {OPERATION_TYPE} from "../../../models/monitoring/operations/operation-type";
+
+angular.module('graphdb.framework.core.directives.operationsstatuses', [])
+    .directive('operationsStatuses', operationsStatusesDirectives);
```
[the remark this answers is not in the document]
Comment (boyan-tonchev) [reply src/js/angular/core/directives/operations-statuses/operations-statuses.directive.js:6]: Ok. I will change the name of directive to operationsStatusMonitor and move all logic from controller into directive.

File: test-cypress/steps/operations-statuses-component-steps.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+    static openOperationStatusesDialog() {
+        OperationsStatusesComponentSteps.getOperationsStatusesComponent().click();
+    }
+
+    static clickOnOperationStatus(operationIndex = 0) {
+        OperationsStatusesComponentSteps.getOperationStatuses().eq(operationIndex).then(($operationElement) => {
+            expect($operationElement).to.have.attr('target', '_blank');
+            // update attr to open in same tab
+            $operationElement.attr('target', '_self');
```
Comment (svilenvelikov) [inline test-cypress/steps/operations-statuses-component-steps.js:19]: I believe that this is not necessarry. You can just verify the href and target attributes and that's it.

File: src/js/angular/core/directives/operations-statuses/operations-statuses.directive.js (hunk 1 above)
Comment (boyan-tonchev) [reply src/js/angular/core/directives/operations-statuses/operations-statuses.directive.js]: Done

File: src/js/angular/controllers.js (hunk 2 above)
Comment (boyan-tonchev) [reply src/js/angular/controllers.js]: Refactored

File: test-cypress/steps/operations-statuses-component-steps.js (hunk 4 above)
Comment (boyan-tonchev) [reply test-cypress/steps/operations-statuses-component-steps.js:19]: Refactored

File: src/js/angular/core/directives/operations-statuses/operations-statuses.directive.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (boyan-tonchev) [reply src/js/angular/core/directives/operations-statuses/operations-statuses.directive.js]: Refactored

**Review by yordanalexandrov — APPROVED** [review]

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**